tests/e2e-upgrade/upgrade-test/01-upgrade-operator.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
-  - command: kubectl apply -f ../../_build/manifests/01-opentelemetry-operator.yaml
+  - script: cd ../../../ && make deploy
   - command: kubectl rollout status -w deployment/opentelemetry-operator-controller-manager -n opentelemetry-operator-system
-  - command: sleep 60s
+  - command: sleep 5s
```

Comment:
This does not seem necessary, the following command waits up to 5 mins.

Reply:
It is because, if the command is executed too fast, the previous operator will be deployed and ready and the rollout will not be completed before this command exits